charts/s3gw/templates/deployment.yaml

```diff
@@ -27,26 +27,43 @@ spec:
           imagePullPolicy: {{ default "IfNotPresent" .Values.imagePullPolicy }}
           args:
             - "--rgw-dns-name"
-            - $(RGW_DNS_NAME)
+{{- if .Values.ingress.enabled }}
+            - {{ .Values.serviceName }}.{{ .Values.publicDomain }},
+              {{ .Values.serviceName }}.{{ .Release.Namespace }}.{{ .Values.privateDomain }}
+{{- else}}
+            - {{ .Values.serviceName }}.{{ .Release.Namespace }}.{{ .Values.privateDomain }}
+{{- end }}
             - "--rgw-backend-store"
-            - $(RGW_BACKEND_STORE)
+            - sfs
             - "--debug-rgw"
-            - $(DEBUG_RGW)
+            - '{{ .Values.logLevel }}'
+            - "--rgw_frontends"
+            - "beast port=7480 ssl_port=7481
+                ssl_certificate=/s3gw-cluster-ip-tls/tls.crt
+                ssl_private_key=/s3gw-cluster-ip-tls/tls.key"
+            - "--rgw_thread_pool_size"
+            - "512"
```

Comment:
Why repeat the default setting here?

Reply:
Right point, I've added it because I was unsure if we should expose this setting to the user or not, so for the time being, I simply added the explicit default value.
What do you think?

Reply:
Restating the default could hunt us in the future. Especially restating it without a comment why we explicitly set this value to 512. 
Doesn't commandline args have higher precedence than ceph.conf?  

I think a better place for this would be a configmap and a file mapping of that to ceph.conf. Ideally having documentation comments on why this value is set

Reply:
ack, I will remove this, we will eventually think better what we want to do this this param